terraform/validators.tf

```diff
@@ -161,6 +161,31 @@ resource "aws_instance" "validator" {
     NodeIndex = count.index
   }
 
+  provisioner "remote-exec" {
+    inline = [
+      "sudo mkdir -p /opt/logstash",
+      "sudo chown -R ec2-user /opt/logstash",
+    ]
+
+    connection {
```

Comment:
@mgorven Is there a permission I need to add somewhere? I tried to provision a file in validator instance, but seems the connection can't be made and timed out

Reply:
TF can't SSH to validator instances because of the jumphost. Add this in the user-data script instead.